src/Core/Algorithms/Legacy/Fields/RegisterWithCorrespondences.cc

```diff
@@ -894,15 +976,18 @@ bool RegisterWithCorrespondencesAlgo::runN(FieldHandle input, FieldHandle Cors1,
   if (!output)
   {
     error("Could not allocate output field");
-    return (false);
+    return nullptr;
   }
 
   VMesh* imesh = input->vmesh();
   VMesh* omesh = output->vmesh();
 
   omesh = imesh;
+  
+  DenseMatrix transArray(Eigen::MatrixXd::Identity(4,4));
+  DenseMatrixHandle transform(new DenseMatrix(transArray.matrix()));
```

Comment:
This can be done on one line without copying:
`return boost::make_shared<DenseMatrix>(std::move(Eigen::MatrixXd::Identity(4,4)));`
If that doesnt work, this is one line that will (maybe) copy (but psha, 16 elements)
`return boost::make_shared<DenseMatrix>(Eigen::MatrixXd::Identity(4,4));`